storage-operator/pkg/salt/client.go

```diff
@@ -95,25 +95,35 @@ func (self *Client) PrepareVolume(
 // Arguments
 //     ctx:      the request context (used for cancellation)
 //     nodeName: name of the node where the volume will be
+//     volumeName: name of the volume to prepare
+//     saltenv:    saltenv to use
 //
 // Returns
 //     The Salt job ID.
 func (self *Client) UnprepareVolume(
-	ctx context.Context, nodeName string,
+	ctx context.Context, nodeName string, volumeName string, saltenv string,
```

Comment:
I'm surprised `go fmt` or `go vet` doesn't turn this into `ctx context.Context, nodeName, volumeName, saltenv string`

Reply:
Indeed, maybe it's not a good practice or not good enough to be enforced.